src/services/user/hooks/userService.js

```diff
@@ -15,7 +15,9 @@ const {
 } = require('../../../utils');
 
 const constants = require('../../../utils/constants');
-const { CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS, SC_DOMAIN } = require('../../../../config/globals');
+const {
+	CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS, SC_DOMAIN, HOST, SC_SHORT_TITLE,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['SC_SHORT_TITLE' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5424589)